src/Sema.zig

```diff
@@ -22722,7 +22719,16 @@ fn zirPtrFromInt(sema: *Sema, block: *Block, inst: Zir.Inst.Index) CompileError!
             .storage = .{ .elems = new_elems },
         } }));
     }
+    if (try sema.typeRequiresComptime(ptr_ty)) {
+        return sema.failWithOwnedErrorMsg(block, msg: {
+            const msg = try sema.errMsg(block, src, "cannot get a runtime pointer to comptime-only type '{}'", .{ptr_ty.fmt(mod)});
```

Comment:
```suggestion
            const msg = try sema.errMsg(block, src, "pointer to comptime-only type '{}' must be comptime-known, but operand is runtime-known", .{ptr_ty.fmt(mod)});
```

Reply:
It's a bit verbose but I do like it better.